container/features/src/main/resources/features.xml

```diff
@@ -1808,6 +1808,9 @@
         <feature>guava</feature>
         <feature>json-patch</feature>
         <bundle>mvn:com.google.protobuf/protobuf-java/${protobufVersion}</bundle>
+        <bundle dependency="true">mvn:com.fasterxml.jackson.core/jackson-databind/${jackson2Version}</bundle>
```

Comment:
`json-patch` feature already have these dependencies. You may not need these dependencies here.